docs/usage/logging.rst

```diff
@@ -32,6 +32,55 @@ Application and request level loggers can be configured using the :class:`~lites
     is keyed as ``queue_listener`` in the logging configuration. The above example is using this handler,
     which is optimal for async applications. Make sure to use it in your own loggers as in the above example.
 
+
+
+Standard Library Logging
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+`logging <https://docs.python.org/3/howto/logging.html>`_ is Python's builtin standard logging library. Works with gunicorn and uvicorn as well.
+
+.. code-block:: python
+
+    from litestar import Litestar, Request, get
+    import logging
+
+
+    def get_logger(module_name: str) -> logging.Logger:
+        """Return logger object."""
+        # create logger
+        logger = logging.getLogger(module_name)
+        logger.setLevel(logging.INFO)
+
+        # create console handler and set level to debug
+        ch = logging.StreamHandler()
+        ch.setLevel(logging.INFO)
+
+        # create formatter
+        formatter = logging.Formatter(
+            "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        )
+
+        # add formatter to ch
+        ch.setFormatter(formatter)
+
+        # add ch to logger
+        logger.addHandler(ch)
+        return logger
+
+
+    logger = get_logger(__name__)
+
+
+    @get("/")
+    def my_router_handler(self, request: Request) -> None:
+        logger.info("inside a request")
+        return None
+
+
+    app = Litestar(route_handlers=[my_router_handler])
+
+
+
```

Comment:
I think we should also include an example of how to use the `LoggingConfig`.  This would be the preferred method for using the logger.

```python
import logging

from litestar import Litestar, Request, get
from litestar.logging.config import LoggingConfig

log_config = LoggingConfig(
    root={"level": logging.getLevelName(logging.INFO), "handlers": ["console"]},
    formatters={"standard": {"format": "%(asctime)s - %(name)s - %(levelname)s - %(message)s"}},
)

logger = log_config.configure()()


@get("/")
def my_router_handler(request: Request) -> None:
    request.logger.info("inside a request")
    logger.info("here too")


app = Litestar(
    route_handlers=[my_router_handler],
    logging_config=log_config,
)
```

Reply:
Ah nice, the above works with Python3.12 and uvicorn. I'm happy to just replace my example with this one actually since it uses the litestar LoggingConfig and works as expected. 

What do you think? The other idea would be to take elements of your example and incorporate them into the very first (not yet changed in this PR) logging example so that the first example users see is one that works on 3.12 and uvicorn?